src/Microsoft.Sbom.Api/Utils/Constants.cs

```diff
@@ -28,6 +33,22 @@ public static class Constants
         Version = "1.0.0"
     };
 
+    public static SbomSpecification SPDX22Specification = ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo);
+
+    public static Collection<SbomSpecification> SupportedSbomSpecifications = new()
+    {
+        ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo),
+        ManifestInfo.ToSBOMSpecification(SPDX30ManifestInfo),
+        ManifestInfo.ToSBOMSpecification(TestManifestInfo),
+    };
+
+    public static Collection<ManifestInfo> SupportedSpdxManifests = new()
+    {
+        SPDX22ManifestInfo,
+        SPDX30ManifestInfo,
+        TestManifestInfo,
```

Comment:
Similar to the above comment--we should not be allowing the test config in the shipping code

Reply:
This actually does not affect the unit tests. I added it as a precaution for future unit tests but didn't realize it couldn't be shipped.